LayoutTests/TestExpectations

```diff
@@ -4828,13 +4828,14 @@ webanimations/translate-property-and-translate-animation-with-delay-on-forced-la
 
 # CSS containment tests that fail
 imported/w3c/web-platform-tests/css/css-contain/contain-inline-size-bfc-floats-001.html [ ImageOnlyFailure ]
-# forced layout
-imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visibility-035.html [ Skip ]
+# can't work since scroll offset is stored in renderer
+imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visibility-081.html [ Skip ]
+# problematic test crbug.com/1247844
```

Comment:
Nit: You might want to write a bit more here about why this test is failing on WebKit. That is, comment on the fact that this is due to scroll offsets being stored in renderers.

Reply:
I think the comment covers it, or do you mean write more about t in the commit log?